packages.ini

```diff
@@ -895,6 +895,8 @@ python_versions = <3.11
 
 [trio-websocket==0.9.2]
 
+[typed-ast==1.5.4]
```

Comment:
typed-ast should not be needed for python3.8+

Reply:
@asottile-sentry this is from `dumpster` (one of our VMs) where we don't have 3.8: 

```
Collecting typed-ast>=1.4.2; python_version < "3.8" and implementation_name == "cpython" (from black==22.6.0->-r k8s/cli/requirements.txt (line 4))
Could not install packages due to an EnvironmentError: 404 Client Error: Not Found for url: https://pypi.devinfra.sentry.io/simple/typed-ast/
```

Reply:
```
mwarkentin in dumpster in ops on  master [?] via 🐍 v3.7.3 (.ops-venv) took 2s
❯ python --version
Python 3.7.3
```

Reply:
sentry-pypi's minimum support is python 3.8 -- should probably find a way to get 3.8 then -- perhaps via https://github.com/getsentry/prebuilt-pythons (I thought josh had set this up already for ops?)